### PR TITLE
Changelog notes for v5.30 / v5.29 backports

### PR DESCRIPTION
**Pull request by amyblais**
for https://mattermost.atlassian.net/browse/MM-31973

### CODE REVIEW

File: source/administration/changelog.md [hunk 1]
```diff
@@ -80,6 +80,8 @@ Multiple setting options were added to ``config.json``. Below is a list of the a
 
 ## Release v5.30 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
 
+- **v5.30.2, released 2021-01-16**
+  - Fixed an issue where a column in the ``ThreadMemberships`` table was missing. Admins planning to enable [Collapsed Reply Threads](https://mattermost.com/blog/dev-sneak-peek-collapsed-reply-threads/) (available in beta in Q1 2021) are recommended to upgrade to v5.30.2 or later.
```
Comment (esethna) [inline source/administration/changelog.md:84]: Fixed an issue where the ``UnreadMentions`` column was missing in the ``ThreadMemberships`` for servers upgrading from an earlier version. Admins planning to enable [Collapsed Reply Threads](https://mattermost.com/blog/dev-sneak-peek-collapsed-reply-threads/) (available in beta in Q1 2021) are recommended to upgrade to v5.30.2 or later.

**Review by esethna — CHANGES_REQUESTED** [review]

**Review by reflog — APPROVED** [review]

File: source/administration/changelog.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Mattermost v5.31.0 contains a low level security fix. [Upgrading](https://docs.m
  - Added a new ``manage_remote_clusters`` permission.
 
 ### Bug Fixes
- - Fixed an issue where a column in the ``ThreadMemberships`` table was missing.
+ - Fixed an issue where the ``UnreadMentions`` column was missing in the ``ThreadMemberships`` table for servers upgrading from an earlier version. Admins planning to enable [Collapsed Reply Threads](https://mattermost.com/blog/dev-sneak-peek-collapsed-reply-threads/) (available in beta in Q1 2021) are recommended to upgrade to v5.31.0 or later.
```
Comment (esethna) [inline source/administration/changelog.md:34]: ...was missing in the ``ThreadMemberships`` table for servers upgrading from Mattermost Server v5.29.0

File: source/administration/changelog.md [hunk 3]
```diff
@@ -80,6 +80,8 @@ Multiple setting options were added to ``config.json``. Below is a list of the a
 
 ## Release v5.30 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
 
+- **v5.30.2, released 2021-01-16**
+  - Fixed an issue where the ``UnreadMentions`` column was missing in the ``ThreadMemberships`` table for servers upgrading from an earlier version. Admins planning to enable [Collapsed Reply Threads](https://mattermost.com/blog/dev-sneak-peek-collapsed-reply-threads/) (available in beta in Q1 2021) are recommended to upgrade to v5.30.2 or later.
```
Comment (esethna) [inline source/administration/changelog.md:84]: ...was missing in the ``ThreadMemberships`` table for servers upgrading from v5.29.0

File: source/administration/changelog.md [hunk 4]
```diff
@@ -188,6 +190,8 @@ Multiple setting options were added to ``config.json``. Below is a list of the a
 
 ## Release v5.29 - [Quality Release](https://handbook.mattermost.com/operations/research-and-development/product/release-process/release-overview)
 
+- **v5.29.2, released 2021-01-16**
+  - Fixed an issue where the ``UnreadMentions`` column was missing in the ``ThreadMemberships`` table for servers upgrading from an earlier version. Admins planning to enable [Collapsed Reply Threads](https://mattermost.com/blog/dev-sneak-peek-collapsed-reply-threads/) (available in beta in Q1 2021) are recommended to upgrade to v5.29.2 or later.
```
Comment (esethna) [inline source/administration/changelog.md:194]: ...was missing in the ``ThreadMemberships`` table for servers upgrading from v5.29.0

**Review by esethna — CHANGES_REQUESTED** [review]

**Comment by esethna** [comment]
@amyblais see the suggestions above since the issue is less severe than originally thought (it only affects servers upgrading from exactly v5.29.0)

**Review by esethna — APPROVED** [review]

**Pull request merged**